dynamiqs/utils/wigners.py

```diff
@@ -45,17 +45,29 @@ def wigner(
     """
     state = jnp.asarray(state)
 
-    if state.ndim > 2:
-        raise NotImplementedError('Batching is not yet implemented for `wigner`.')
-
     rdtype = dtype_complex_to_real(state.dtype)
     xvec = jnp.linspace(-xmax, xmax, npixels, dtype=rdtype)
     yvec = jnp.linspace(-ymax, ymax, npixels, dtype=rdtype)
 
     if method == 'clenshaw':
         state = todm(state)
-        w = _wigner_clenshaw(state, xvec, yvec, g)
+        expanded = False
+        if state.ndim == 2:
+            state = state[None, ...]
+            expanded = True
+
+        n = state.shape[-1]
+        w = jax.vmap(_wigner_clenshaw, in_axes=(0, None, None, None, None))(
```

Comment:
```suggestion
        w = jax.vmap(_wigner_clenshaw, in_axes=(-3, None, None, None, None))(
```
Can you try and see if it works on doubly-batched arrays? (e.g. `(8, 5, 16, 16)`)

Reply:
I'm not sure what you want to do with `-3` 🤔 
It would work with a double vmap

Maybe jitting wigner direclty would be better, I'm going to see what we can do

Reply:
I thought "iterate over the third axis starting from last of the third element", to support `(..., n, n)` and not just `(b, n, n)`, but it's maybe not the correct way to do. We want the function to work with arbitrarily batched input.